dustmite.d

```diff
@@ -1312,20 +1312,20 @@ void dumpSet(string fn)
 				"[",
 				e.noRemove ? "!" : "",
 				" ",
-				e.isFile ? e.filename ? printableFN(e.filename) ~ " " : null : e.head ? printable(e.head) ~ " " : null,
-				e.tail ? printable(e.tail) ~ " " : null,
-				e.comment ? "/* " ~ e.comment ~ " */ " : null,
+				e.isFile ? e.filename !is null ? printableFN(e.filename) ~ " " : null : e.head !is null ? printable(e.head) ~ " " : null,
+				e.tail !is null ? printable(e.tail) ~ " " : null,
+				e.comment !is null ? "/* " ~ e.comment ~ " */ " : null,
```

Comment:
Nevermind, I didn't see what printable and printableFN do, indeed they add extra things, even for blank strings. Sorry for the noise.